flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -380,6 +389,53 @@ public void testTableSourceCommitOnCheckpointDisabled() {
                                 .noDefaultValue()));
     }
 
+    @ParameterizedTest
+    @MethodSource("offsetResetStrategy")
```

Comment:
I do not think the method source is necessary here you can use the following:

```java
    @ValueSource(strings = {"none", "earliest", "latest"})
    @NullSource
```

Reply:
fixed